data/json/items/generic.json

```diff
@@ -359,7 +359,7 @@
     "symbol": "o",
     "color": "light_gray",
     "name": { "str": "cluster of gas sacs", "str_pl": "clusters of gas sacs" },
-    "description": "This is a cluster of membranous bubbles, each about the size of a grape, retrieved from inside a mutant insect.  They float like tiny helium balloons, and are likely full of a lighter-than-air gas helping the bug to fly.",
+    "description": "This is a cluster of membranous bubbles, each about the size of a grape, retrieved from inside a mutant insect.  They float like tiny helium balloons.  Perhaps they are full of a lighter-than-air gas.",
```

Comment:
```suggestion
    "description": "This is a cluster of membranous bubbles, each about the size of a grape, retrieved from inside a mutant insect.  They float like tiny helium balloons - perhaps they are full of a lighter-than-air gas?",
```
Flows better IMO.

Reply:
I like the hyphen but I kept the question phrasing out, again to avoid implying player thought too deeply.